services/core-api/app/api/mines/response_models.py

```diff
@@ -81,29 +80,31 @@ def format(self, value):
     })
 
 PERMIT_AMENDMENT_MODEL = api.model(
-    'PermitAmendment', {
-        #'permit_guid':fields.String,
-        'permit_amendment_id':fields.Integer,
+    'PermitAmendment',
+    {
+                                                                                         #'permit_guid':fields.String,
+        'permit_amendment_id': fields.Integer,
         'permit_amendment_guid': fields.String,
         'permit_amendment_status_code': fields.String,
         'permit_amendment_type_code': fields.String,
         'received_date': fields.DateTime(dt_format='iso8601'),
         'issue_date': fields.DateTime(dt_format='iso8601'),
         'authorization_end_date': fields.DateTime(dt_format='iso8601'),
-        #'permit_amendment_status_description': fields.String,
-        #'permit_amendment_type_description': fields.String,
+                                                                                         #'permit_amendment_status_description': fields.String,
```

Comment:
This indenting seems way off for some reason.

Reply:
I agree. It was the auto-formatter. Maybe there is a setting we can add to prevent this.